conf/messages

```diff
@@ -776,6 +776,26 @@ calc.summary.whatToDoNext.details = Before you continue, save a copy of your cal
 calc.summary.noticeSummary = Your result may be slightly inaccurate because the calculator does not support the dates you entered. Do not use these figures to report your Capital Gains Tax.
 calc.summary.download = Download
 
+## What to do next
+calc.whatToDoNext.title = What to do next
+calc.whatToDoNext.finish = Finish
+calc.whatToDoNext.reportNow = Report now
+
+calc.whatToDoNext.fourAEA.information = You have until {0} to report your Capital Gains Tax figures through Self Assessment.
```

Comment:
Please change to `You''ll have to report your Capital Gains Tax figures through Self Assessment.`